ctest/test_image.c

```diff
@@ -32,60 +32,14 @@ float *uniformf(float a, float b, int n)
 {
   int i;
   float *result;
-    
+
   result = (float*)malloc(n*sizeof(float));
   for (i=0; i<n; i++)
     result[i] = a + (b-a) * rand() / ((double)RAND_MAX);
 
   return result;
 }
 
-double gaussrand()
```

Comment:
Ah yes, I also removed this because while looking for `static`s in the codebase that need to be updated for thread-safety, I've noticed that those functions are unused altogether, so just removed them.